src/components/organisms/ChannelInfoForm/ChannelInfoForm.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from 'react';
+import { Grid, makeStyles } from '@material-ui/core';
+import Input from '../../atoms/Input/Input';
+import Switch from '../../atoms/Switch/Switch';
+import Typo from '../../atoms/Typo/Typo';
+import Button from '../../atoms/Button/Button';
+
+const useStyles = makeStyles({
+  root: {
+    padding: '5em 0',
+    width: '300px',
+  },
+  margin: {
+    marginBottom: '1em',
+  },
+  button: {
+    marginTop: '1.68em',
+    width: '32.8%',
+  },
+});
+
+// eslint-disable-next-line no-unused-vars
+type ChannelInfoFormProps = { setOpen: (isOpen:boolean) => void };
+
+const ChannelInfoForm = ({ setOpen }: ChannelInfoFormProps) => {
+  const [channelName, setChannelName] = useState<string>('');
+  const [isValidChannelName, setValidChannelName] = useState<boolean>(false);
+  const [helperTextChannelName, setHelperTextChannelName] = useState<string>('영문+숫자 5-25자');
```

Comment:
```suggestion
  const CHANNEL_HELPER_TEXT = '영문+숫자 5-25자';
  const [helperTextChannelName, setHelperTextChannelName] = useState<string>(CHANNEL_HELPER_TEXT);
```
이 string의 내용(`영문+숫자 5-25자`) 자체는 유동적으로 변하는 게 아니라 기본적으로 같은 값을 가지고 있으니 literal으로 매번 작성하는 대신 이렇게 constant로 따로 빼서 재활용해도 괜찮을 것 같습니다! (반복되어 쓰이는 다른 helperText들도 동일) 이 방식으로 수정하게 된다면 이 constant 자체는 component 바깥에 있어도 괜찮을 것 같아요. 
UserInfoForm 작성할 때는 생각을 못 했던 부분인데, 중복이 많이 보이니까 이렇게 하면 더 좋겠다는 생각이 드네요! 이 브랜치 내용은 아니지만 나중에 UserInfoForm도 이런 식으로 constant 이용해서 이렇게 수정하면 좋을 것 같습니다. ㅎㅎ

Reply:
네 확인했습니다! 상수로 하는 것이 좀 더 깔끔한 것같습니다. 그렇게 수정하겠습니다!

Reply:
해당 코드 수정했습니다! 감사합니다!
```
const CHANNEL_NAME_AVAILABLE = '사용할 수 있는 채널이름';
const CHANNEL_NAME_HELPER_TEXT = '영문+숫자 5-25자';
const CHANNEL_PASSWORD_AVAILABLE = '사용할 수 있는 비밀번호';
const CHANNEL_PASSWORD_HELPER_TEXT = '영문+숫자 4-12자';
const PASSWORD_CHECK_YES = '비밀번호 일치';
const PASSWORD_CHECK_NO = '비밀번호 일치하지 않음';
```